docs/conf.py

```diff
@@ -25,9 +25,7 @@
 # -- Options for HTML output -------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 
-html_theme = 'insegel'  # Clean black-on-white theme
-if os.environ.get('TBPULUMI_DARK_MODE', False):
-    html_theme = 'furo'  # Dark theme, easy on the eyes
+html_theme = 'furo'  # Clean, organized theme with dark and light modes
```

Comment:
I'm okay with furo, but our other readthedocs pages use the standard boring readthedocs theme. (In tb-accounts case mermaid extension doesn't mesh well with furo without some tweaks.) Should we discuss standardizing this in the next code / infra meeting?

Reply:
We can. It's easy enough to swap out to that theme. The lack of a dark mode drives me bonkers. My eyes can't handle all those white pixels. Totally up for discussion during an infra meeting.

Reply:
no worries, I also hate light themes! It's kind of bonkers that the readthedocs theme doesn't have a dark mode via prefers-color-scheme.